app/Resources/translations/default/AdminCatalogFeature.xlf

```diff
@@ -1367,9 +1367,9 @@ File: src/PrestaShopBundle/Translation/Api/StockApi.php:48</note>
         <note>Context:
 File: src/PrestaShopBundle/Translation/Api/StockApi.php:49</note>
       </trans-unit>
-      <trans-unit id="dc7f9d531239c6acc08b99236a4204cd">
-        <source>Stock management</source>
-        <target>Stock management</target>
+      <trans-unit id="d164dc7a9ce1ca081fcee017f54b21ec">
+        <source>Stocks management</source>
```

Comment:
Sorry, late to the party, but are we sure about that?
It seems to me that "[Stock management](https://en.wikipedia.org/wiki/Stock_management)" should be singular.
And I wonder about the rest of these plurals. @AlexEven, can you chime in?